view/templates/content/image/single_with_height_allocation.tpl

```diff
@@ -2,7 +2,7 @@
 	As a result, we need to add a wrapping element for non-flex (non-image grid) environments, mostly single-image cases.
  *}}
 {{if $allocated_max_width}}
-<div style="max-width: {{$allocated_max_width|default:"auto"}};">
+<div class="allocated-max-width" style="max-width: {{$allocated_max_width|default:"auto"}};">
```

Comment:
Please follow the naming convention of the below class `img-allocated-height`:
```suggestion
<div class="img-allocated-max-width" style="max-width: {{$allocated_max_width|default:"auto"}};">
```

Reply:
should be done